lumigator/python/mzai/jobs/inference/inference.py

```diff
@@ -0,0 +1,161 @@
+"""python job to run batch inference"""
+
+import argparse
+import functools
+import json
+import os
+import time
+from collections.abc import Iterable
+from pathlib import Path
+
+import s3fs
+from box import Box
+from datasets import load_from_disk
+from loguru import logger
+from model_clients import (
+    BaseModelClient,
+    MistralModelClient,
+    OpenAIModelClient,
+)
+
+# SummarizationPipelineModelClient,
+# HuggingFaceModelClient,
+from tqdm import tqdm
+
+
+def timer(func):
```

Comment:
are we still thinking we want this given job logging in Ray?

Reply:
Not sure about that: it is convenient now that the code is still in development but I think it was more useful in the eval script (that returned a json including these metrics) rather than here (where we'll likely just return a CSV file with one additional column)

Reply:
in that case, let's remove?

Reply:
Done!